### PR TITLE
edits on README.me for setup on virtualenv

### PR DESCRIPTION
**Pull request by goweiting**
Tested the previous instructions from README.me on Ubuntu 16.04 LTS;
added some additional comments and steps to configure the carneades software.

### CODE REVIEW

**Comment by ewan-klein** [comment]
Thanks @goweiting. @edran, does this look reasonable to you?

**Comment by edran** [comment]
Don't have the time to test on a clean machine. @goweiting did anybody else successfully install this on a clean 16.04 machine following your instructions? If so, LGTM :)

**Comment by goweiting** [comment]
@edran I havent solicited feedback from the class yet. The changes helps to troubleshoot the errors during installation.

File: README.md [hunk 1]
```diff
@@ -61,33 +61,53 @@ because of some error, it's likely to be one of the following
...
 
 If you are using a virtualenv or you don't use Ubuntu, do the following:
 
-* Download and extract the pycairo package in the virtualenv
+* Download and extract the `pycairo` package _in_ the virtualenv
 
 ```bash
 $ curl http://cairographics.org/releases/pycairo-1.10.0.tar.bz2 -O
 $ tar xvf pycairo-1.10.0.tar.bz2
+# That doesnt quite work for me, but these do:
```
Comment (goweiting) [inline README.md]: Another friend had problem installing by curl-ing too.

**Comment by ewan-klein** [comment]
Hi @goweiting. Many thanks for doing this. Apologies for not doing the merge sooner.

**Pull request merged**